File: app-java/src/main/java/com/paytpv/appjava/utils/CardHandler.java
package com.paytpv.appjava.utils;

import com.paytpv.androidsdk.Model.Basic.PTPVCard;

import java.util.ArrayList;

public class CardHandler {
    private static CardHandler singleton = new CardHandler( );
    private ArrayList<PTPVCard> cardArray = new ArrayList<>();

    private CardHandler() { }

    public static CardHandler getInstance( ) {
        return singleton;
    }

    public ArrayList<PTPVCard> getCardArray() {
        return cardArray;
    }
}

File: app-kotlin/src/main/java/com/paytpv/paytpv/utils/CardHandler.kt
package com.paytpv.paytpv.utils

import com.paytpv.androidsdk.Model.Basic.PTPVCard

class CardHandler private constructor() {

    private object Holder {
        val INSTANCE = CardHandler()
    }

    companion object {
        val instance: CardHandler by lazy { Holder.INSTANCE }
    }

    var cardArray: ArrayList<PTPVCard>? = ArrayList()
}
